MarsMission/Models/WeatherData.swift

```diff
@@ -8,19 +8,15 @@
 import Foundation
 
 struct WeatherData: Codable {
-	var forecasts: [forecastsData]
+	var forecasts: [forecast]
```

Comment:
class Forecast should be named with a capital. Remember that all classes in swift are named with a capital